internal/ingress/annotations/auth/main.go

```diff
@@ -130,7 +141,7 @@ func (a auth) Parse(ing *extensions.Ingress) (interface{}, error) {
 		File:    passFile,
 		Secured: true,
 		FileSHA: file.SHA1(passFile),
-		Secret:  name,
+		Secret:  sname,
```

Comment:
With https://github.com/kubernetes/ingress-nginx/pull/3875/files#r264246598, this change is not needed

Reply:
done